tests/10-create.js

```diff
@@ -0,0 +1,167 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
+ */
+import {shouldBeBs58, shouldBeMulticodecEncoded} from './assertions.js';
+import chai from 'chai';
+import {
+  checkDataIntegrityProofFormat
+} from 'data-integrity-test-suite-assertion';
+import {createInitialVc} from './helpers.js';
+import {documentLoader} from './documentLoader.js';
+import {endpoints} from 'vc-api-test-suite-implementations';
+import {validVc as vc} from './validVc.js';
+
+const tag = 'ecdsa-2019';
+const cryptosuite = 'ecdsa-2019';
+const {match, nonMatch} = endpoints.filterByTag({
+  tags: [tag],
+  property: 'issuers'
+});
+const should = chai.should();
+
+describe('ecdsa-2019 (create)', function() {
+  checkDataIntegrityProofFormat({
+    implemented: match,
+    notImplemented: nonMatch
+  });
+  describe('ecdsa-2019 (issuer)', function() {
+    this.matrix = true;
+    this.report = true;
+    this.implemented = [...match.keys()];
+    this.notImplemented = [...nonMatch.keys()];
+    this.rowLabel = 'Test Name';
+    this.columnLabel = 'Implementation';
+    for(const [name, {endpoints, implementation}] of match) {
+      describe(name, function() {
+        const [issuer] = endpoints;
+        const verifier = implementation.verifiers.find(
+          v => v.tags.has(tag));
+        let issuedVc;
+        let proofs;
+        const verificationMethodDocuments = [];
+        before(async function() {
+          issuedVc = await createInitialVc({issuer, vc});
+          proofs = Array.isArray(issuedVc?.proof) ?
+            issuedVc.proof.filter(proof => proof.cryptosuite === cryptosuite) :
+            [issuedVc?.proof];
+          proofs.should.not.eql([], 'Expected at least one proof to have ' +
+            'cryptosuite "ecdsa-2019".');
+          const verificationMethods = proofs.map(
+            proof => proof.verificationMethod);
+          for(const verificationMethod of verificationMethods) {
+            const verificationMethodDocument = await documentLoader({
+              url: verificationMethod
+            });
+            verificationMethodDocuments.push(verificationMethodDocument);
+          }
+        });
+        it('MUST have property "cryptosuite" and be a string.', function() {
+          this.test.cell = {columnId: name, rowId: this.test.title};
+          proofs.forEach(proof => {
+            should.exist(proof.cryptosuite, 'Expected proof to have ' +
+              '"cryptosuite" property.');
+            proof.cryptosuite.should.be.a('string', 'Expected "cryptosuite" ' +
+              'property to be a string.');
+          });
+        });
+        it('The field "cryptosuite" MUST be "ecdsa-2019".', function() {
+          this.test.cell = {columnId: name, rowId: this.test.title};
+          proofs.forEach(proof => {
```

Comment:
this should probably be a `proofs.some` as an issuer could sign a VC with multiple proofs some of which might not be data integrity or ecdsa.

Reply:
Updated, however see my comment above https://github.com/w3c-ccg/vc-di-ecdsa-test-suite/pull/1#discussion_r1254965677